ctms/bin/acoustic_newsletters_mapping.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+"""Manage Acoustic newsletters mapping: add, list, and remove from the db."""
+
+import argparse
+import os
+import sys
+
+from ctms import config
+from ctms.crud import (
+    create_acoustic_newsletters_mapping,
+    delete_acoustic_newsletters_mapping,
+    get_all_acoustic_newsletters_mapping,
+)
+from ctms.database import get_db_engine
+
+
+def main(dbsession, test_args=None) -> int:
+    parser = argparse.ArgumentParser(
+        description="Manage Acoustic Newsletter mapping",
+        usage="""
+        python acoustic_newsletters_mapping.py add "test-pilot:sub_new_test_pilot" """,
```

Comment:
I think we may want to move this help text to the `add` subparser, since currently this help text will display even when we ask for help on the `remove` command.
```
> python ctms/bin/acoustic_newsletters_mapping.py remove --help
usage: python acoustic_newsletters_mapping.py add "test-pilot:sub_new_test_pilot" remove [-h] source

positional arguments:
  source      Remove newsletter mapping with specified 'source'

options:
  -h, --help  show this help message and exit
```